cosmwasm/packages/wasmi-runtime/src/exports.rs

```diff
@@ -181,3 +181,11 @@ pub unsafe extern "C" fn ecall_query(
         }
     }
 }
+
+/// # Safety
+/// Always use protection
+#[no_mangle]
+pub unsafe extern "C" fn ecall_health_check(
+) -> HealthCheckResult {
+    HealthCheckResult::Success
+}
```

Comment:
Is this intentionally empty? You could add some checks here, theoretically.
also, why the newline between the function definition's `()`s?

Reply:
Intentionally empty, yes. When we want to check other things we can add to it, but right now I wanted something that just reports if the enclave is running fine or not